FinancialTracker/SceneDelegate.swift

```diff
@@ -60,14 +62,16 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 
     func sceneWillEnterForeground(_ scene: UIScene) {
-        // Called as the scene transitions from the background to the foreground.
-        // Use this method to undo the changes made on entering the background.
+        startDate = Date.init()
     }
 
     func sceneDidEnterBackground(_ scene: UIScene) {
-        // Called as the scene transitions from the foreground to the background.
-        // Use this method to save data, release shared resources, and store enough scene-specific state information
-        // to restore the scene back to its current state.
+        let timeActive = startDate.timeIntervalSinceNow * -1
```

Comment:
why `* -1`?

Reply:
for some reason the interval is negative, its correct but negative

Reply:
Yes, because current date (`timeIntervalSinceNow`) is a bigger number than the `startDate`

Reply:
Add a comment for that.